### ISSUE TITLE
Hand recognition does not work on macbook air

### ISSUE THREAD

**Issue by ApproximateIdentity** [UnitedLexCorp/SimpleTalk#145]
The camera is not loading right.

**Comment by ApproximateIdentity** [comment]
The issue turned out to be that I was hosting simpletalk on a different laptop from macbook air over the local network and that I was not using https which apparently is not allowed. So in this case easiest was to just portforward 8000 from the macbook to the other laptop to trick the browser into thinking it was local. (Obviously I could also just have run it in on the air, but I didn't feel like getting all the node stuff installed there.)

So basically falso alarm I'm closing this issue.
